model/src/main/java/org/cloudfoundry/identity/uaa/provider/AbstractExternalOAuthIdentityProviderDefinition.java

```diff
@@ -214,6 +215,14 @@ public boolean isPkce() {
         return this.pkce;
     }
 
+    public boolean isPerformRpInitiatedLogout() {
```

Comment:
I know that `isPerform` is a common pattern in the UAA, but we have a chance to make the world a better place here :).

Would you object to calling this method `doPerformRpInitiatedLogout` instead? Or maybe just `performsRpInitiatedLogout`. The object _isn't_ a performed RP initiated logout, it does one, or enables one.

Reply:
Yeah, grammatically, `isPerformRpInitiatedLogout` does not make sense. But what made me feel that this is acceptable:
1. the other boolean configs in the same class follow this pattern:
```
isShowLinkText
isClientAuthInBody
isPkce
```
I don't wanna create an inconsistency within the same class (or I want a refactor to apply to all "getters" like this).
2. intellij's auto getter/setter generation will follow this `isSomeBoolean` pattern, which suggests that this pattern is quite standard. 

Yeah, if this function is a more custom function, I would feel more free to name it however I see fit. But since this is a standard required getter, I feel the need to follow the standard pattern / pattern in the same class for now.